packages/app-cli/app/command-keymaps.ts

```diff
@@ -0,0 +1,47 @@
+import BaseCommand from './base-command';
+import app from './app';
+import { _ } from '@joplin/lib/locale';
+const { cliUtils } = require('./cli-utils.js');
+
+class Command extends BaseCommand {
+	public override usage() {
+		return 'keymaps';
```

Comment:
```diff
-		return 'keymaps';
+		return 'keymap';
```

**Optional**: Consider renaming from `keymaps` to `keymap` (or to `keybinds`) to match the pull request title and/or description.

Reply:
Updated as well